docs/image-build.md

```diff
@@ -0,0 +1,24 @@
+
+# Kube-compare image build
+
+## Build container
+
+```bash
+make image-build
```

Comment:
now that we REAME.md updates...we can move under her https://github.com/openshift/kube-compare?tab=readme-ov-file#install ?

Reply:
Maybe adding a link here? https://github.com/openshift/kube-compare?tab=readme-ov-file#further-resources

Reply:
Sounds good

Reply:
Done